bindings/stronghold-nodejs/js/stronghold.ts

```diff
@@ -16,10 +16,6 @@ export class Stronghold implements Storage {
         return stronghold
     }
 
-    public async setPassword(encryptionKey: Uint8Array) {
```

Comment:
I assume we still need this to set the stronghold password, although it is not part of the interface anymore.

Reply:
I agree, thanks Eike.

Reply:
It's not required because we set the password once during construction. This does not really suffice in the current stronghold, because the password expires after some time. This is also the case in Rust, though. Once we have migrated to the refactored stronghold, it will be fixed. I still think we can remove `set_password` in the NAPI bindings already.

Reply:
Right, because we set it in the constructor initially, which is enough for the new stronghold version. How do we coordinate this?

Reply:
Coordinate the upgrade? Personally, I'd like to do it as soon as possible once the stronghold team gives the green light.

Reply:
Or maybe worded differently: what is the effect if we merge it like this and do we need for a new stronghold version for a potential dev release?

Reply:
> what is the effect if we merge it like this

Sorry for the delay, had to re-understand how password expiration works. Actually, the default is that the password does not expire at all. Merging it like this is therefore fine, as we set the password during construction, and then we don't need to set it again.



> do we need for a new stronghold version for a potential (dev) release?

I don't think we _need_ it. However, I can't guarantee that there is not going to be a breaking change during the migration to the new version. That said, from doing the `KeyLocation` refactor and from what I know about the new version, at least I cannot say that there will definitely be anything broken after the migration.

Reply:
Thank you for clarifying. I was worried it would expire. SO we can totally merge and do a dev release.
Let's discuss potential breaking changes before 0.5.0 proper release.